nixos/modules/services/databases/postgresql.xml

```diff
@@ -51,9 +43,7 @@ Type "help" for help.
 -->
 
   <para>
-   By default, PostgreSQL stores its databases in
-   <filename>/var/lib/postgresql/$psqlSchema</filename>. You can override this using
-   <xref linkend="opt-services.postgresql.dataDir"/>, e.g.
+   By default, PostgreSQL stores its databases in <filename>/var/lib/postgresql/$psqlSchema</filename>. You can override this using <xref linkend="opt-services.postgresql.dataDir"/>, e.g.
```

Comment:
Any reason this is in one line now? IIRC this doesn't have any effect on the rendered result from docbook and I found the previous version easier to edit.

Reply:
https://github.com/NixOS/nixpkgs/commit/641f6356d387ef493812de23f75114effc0cc398